dts/bindings/rtc/atmel,sam-rtc.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAM family RTC device
+
+compatible: "atmel,sam-rtc"
+
+include: rtc-device.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  peripheral-id:
+    type: int
+    description: peripheral ID
+    required: true
```

Comment:
This is out of date. You should use clocks now.

Reply:
using clocks now